tests/install_testbed/create_mysqldb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
```

Comment:
Delete this line is is not a component that could be executed directly from the CLI.